concent_api/conductor/tests/test_integration_conductor.py

```diff
@@ -1,17 +1,18 @@
 import mock
 
-from django.urls    import reverse
+from django.urls import reverse
 
 from common.helpers import get_current_utc_timestamp
 from common.helpers import get_storage_result_file_path
 from common.helpers import get_storage_source_file_path
 from core.message_handlers import store_subtask
 from core.models import Subtask
 from core.tests.utils import ConcentIntegrationTestCase
-from ..models       import BlenderSubtaskDefinition
-from ..models       import UploadReport
-from ..models       import VerificationRequest
-from ..tasks        import blender_verification_request
+from ..models import BlenderSubtaskDefinition
+from ..models import ResultTransferRequest
+from ..models import UploadReport
+from ..models import VerificationRequest
+from ..tasks import blender_verification_request
```

Comment:
Can't we use `from core.tasks ...` and `from core.models ...`? It would be more readable I think.

Reply:
:+1: done